project/src/main/kotlin/org/ionproject/core/accessControl/pap/sql/AuthRepoImpl.kt

```diff
@@ -31,28 +31,24 @@ class AuthRepoImpl(private val tm: TransactionManager) : AuthRepo {
     private val tokenGenerator = TokenGenerator()
 
     override fun getToken(tokenHash: String, derived: Boolean): TokenEntity? = tm.run { handle ->
-        {
```

Comment:
This were lambdas inside lambdas? How strange. Good catch.

Reply:
The IDE helped to find those :)